examples/reference/widgets/TextInput.ipynb

```diff
@@ -63,7 +63,7 @@
    "source": [
```

Comment:
It possible to trigger the value_input value below. Not always but if you type a bit fast.

<img width="1555" height="336" alt="image" src="https://github.com/user-attachments/assets/789c6501-28ab-46a5-8490-54c7cd76e696" />